.gitignore

```diff
@@ -11,7 +11,6 @@
 *.class
 
 # Generated files
-bin/
```

Comment:
Consider a `build-support` or similar directory, rather than removing `bin` from .gitignore. That way we don't accidentally start committing artifacts at some point in the future when we decide it's a good idea for Amplify to generate binaries :)